Add support for SwiftPM based dependency managers

Jeehut opened the pull request:
This adds support for SwiftPM manifest based dependency managers. Specifically this adds support for installing via [Accio](https://github.com/JamitLabs/Accio) but will probably also work with SwiftPM once it's integrated into Xcode.

Please note that this project is part of Accio's official [integration tests](https://github.com/JamitLabs/Accio/tree/work/1000-frameworks/Demo/Shared/AppDependencies) within the [Demo project](https://github.com/JamitLabs/Accio/tree/work/1000-frameworks/Demo).

Jeehut committed:
Add basic SwiftPM manifest file

Jeehut committed:
[README] Document support & installation via Accio

turtlemaster19 commented:
@hpique This solves #453. Please implement this!

UInt2048 reviewed (approved):
@Jeehut You may want to correct the Swift tools version to 5.0. Not required to be merged, though. ;)

Jeehut committed:
Update to Swift 5.0 & add SwiftPM badge

Jeehut commented:
@turtlemaster19 @hpique I just rebased my PR, updated the Swift version to 5.0, added a badge for SwiftPM support and tested SwiftPM support (it works!).

Note that after merging, you would need to make a new release so users don't have to point to the "master" branch and can use it as documented in the README, which points to a new 0.11.2 release. (Feel free to change the documented version number if you do a 0.12.0 release.)

turtlemaster19 commented:
@Lascorbe or any others, any update on this? I want to use `HanekeSwift` in my repository instead of the Objective-C predecessor, and I would prefer to use the official version than this fork.

turtlemaster19 commented:
@Jeehut Until anyone from Haneke responds, can you create this 0.11.2 release in your repository? It would be much more convenient to have a release there for testing purposes.

fredpi commented:
@joanromano Would it be possible to merge this PR? It shouldn't take much time ;)

joanromano commented:
@fredpi I am really sorry about this taking SO LONG - as you said it was just a matter of seconds, so I take responsibility on that one. My most sincere apologies again, and thanks so much @Jeehut on making this possible.
